canonical-web-and-design/snapcraft.io
# Remove global max-width overrides. Add docs specific override

Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/2312

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004
- Browse the site, make sure headings and paragraphs don't look out of place
- Specifically check the docs pages.
